lib/beaker/host/windows.rb

```diff
@@ -43,6 +43,28 @@ def determine_ssh_server
       @ssh_server
     end
 
+    # Gets the path & file name for the puppet agent dev package on Windows
+    #
+    # @param [String] puppet_collection Name of the puppet collection to use
+    # @param [String] puppet_agent_version Version of puppet agent to get
+    # @param [Hash{Symbol=>String}] opts Options hash to provide extra values
+    #
+    # @note Windows only uses the 'install_32' option of the opts hash at this
+    #   time. Note that it will not fail if not provided, however
+    #
+    # @return [String, String] Path to the directory and filename of the package, respectively
+    def puppet_agent_dev_package_info( puppet_collection = nil, puppet_agent_version = nil, opts = {} )
```

Comment:
no.  This is a method override, so it needs to match the signature of the others, even though Windows doesn't require that information for its use.